### ISSUE TITLE
filter play stats based on formation?

### ISSUE THREAD

**Issue by pwaterm2** [BurntSushi/nflgame#262]
Is there a way to filter play stats based on formation?  Is there something that can go in plays.filter(x) that will give me the formation?  I didn't see anything in statmap.py, so it looks like i'll just have to output all the plays and then sum things up in excel.

The end goal is to get stats for each running back in shotgun formations vs non shotgun formations, something like this:

player, shotgun attempts, shotgun yards, shotgun tds, non-shotgun attempts, non-shotgun yards, non-shotgun tds, etc.

**Comment by BurntSushi** [comment]
I've never seen anything about formations.

**Comment by pwaterm2** [comment]
Yeah I assumed their probably wasn't a way to do it, just thought I would ask-- in the play description it will say "shotgun" before the description of the play.  Thanks
